internal/indexer/indexer.go

```diff
@@ -108,6 +108,7 @@ func (i *Indexer) Index() error {
 	i.emitMetadataAndProjectVertex()
 	i.emitDocuments()
 	i.addImports()
+	i.indexDocumentation()
```

Comment:
Could we enable this via flag since it's currently a non-published extension?

Reply:
what needs to be done to un-flag it in your view? doesn't it being behind a flag mean off-by-default?

Reply:
Off-by-default makes sense today since we don't have a backend that can consume it, so we'd be inflating index time for customers that can't make use of the new data yet. I'd say make it on-by-default (or always on) once we have a version that ships the feature.

For auto-indexing on cloud and for our own code we can put the flag explicitly for the next month or so.

Reply:
My aim is to only merge this PR once I have the actual Sourcegraph backend consuming this data and optionally rendering it behind a feature flag in the site config. Given that, do you still think it should be off-by-default here too (two feature flags, effectively)?

> For auto-indexing on cloud and for our own code we can put the flag explicitly for the next month or so.

Where would this be done? I'm not familiar with e.g. where auto indexing gets invoked.

Reply:
Actually if we plan to consume it before we merge this I'm fine just emitting it always.